README.md

```diff
@@ -1,17 +1,74 @@
 # klaviyo-android-sdk
 
-## This project is still in pre-alpha
-## Breaking changes are still being made to the API. This is not yet intended for use
+## DISCLAIMER
+*This project is still in pre-alpha,
+breaking changes are still being made to the API. 
+This is not yet intended for public use*
+
+Android SDK allows developers to incorporate Klaviyo event and profile tracking functionality
+within native Android applications.
+The SDK assists in identifying users and tracking user events.
+Once integrated, your marketing team will be able to better understand your app users' needs and
+send them timely messages via FCM.
+
+## Installation
+*TODO*
 
 ## Core SDK
 
-Android SDK allows developers to incorporate Klaviyo event and profile tracking functionality within native Android applications.
+### Configuration
+The SDK must be configured with the public API key for your Klaviyo account.
+It also requires application context so the SDK can be responsive to 
```

Comment:
`It also requires the application context`?

Reply:
I don't love this sentence but I was just trying to explain why a required argument of `Klaviyo.configure` is the application's `Context` object.

Reply:
Maybe better if I at least put that in backticks like
> We require access to the application `Context` so....

Reply:
Ya I like the sound of that. But maybe `application Context` in backticks rather than just `context`?

Reply:
`applicationContext` is the property's name, `Context` is the type name... but yeah `applicationContext` reads best.